packages/runner/lib/register.ts

```diff
@@ -12,27 +11,6 @@ export async function register(): Promise<Result<void>> {
         return Err('RUNNER_URL is not set');
     }
     try {
-        // In Render, network configuration can take up to 30 seconds to be applied to the service
-        // so we need to wait for the runner to be reachable from the outside before registering
-        const startTime = Date.now();
-        const timeoutMs = 120_000;
-        const waitMs = 1000;
-        let isReachable = false;
-        while (!isReachable && Date.now() - startTime < timeoutMs) {
-            try {
-                const res = await fetch(`${envs.RUNNER_URL}/health`);
-                if (res.ok) {
-                    isReachable = true;
-                } else {
-                    await setTimeout(waitMs);
-                }
-            } catch {
-                await setTimeout(waitMs);
-            }
-        }
-        if (!isReachable) {
-            return Err(new Error(`Runner at ${envs.RUNNER_URL} is not reachable after ${timeoutMs}ms`));
-        }
```

Comment:
we were previously waiting in the runner itself but it looks like it is possible for the runner to reach itself before it is actually reachable from jobs :(